sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -637,11 +644,25 @@ def download_blob(self, offset=None, length=None, **kwargs):
     def _quick_query_options(self, query_expression,
                              **kwargs):
         # type: (str, **Any) -> Dict[str, Any]
-        input_serialization = kwargs.pop('input_serialization', None)
-        output_serialization = kwargs.pop('output_serialization', None)
-        query_request = QueryRequest(expression=query_expression,
-                                     input_serialization=get_quick_query_serialization_info(input_serialization),
-                                     output_serialization=get_quick_query_serialization_info(output_serialization))
+        delimiter = '\n'
+        input_format = kwargs.pop('blob_format', None)
+        if input_format:
+            try:
+                delimiter = input_format.lineterminator
+            except AttributeError:
+                delimiter = input_format.delimiter
+        output_format = kwargs.pop('output_format', None)
+        if output_format:
+            try:
+                delimiter = output_format.lineterminator
+            except AttributeError:
+                delimiter = output_format.delimiter
+        has_header = kwargs.pop('has_header', None)
```

Comment:
should this be false by default

```suggestion
        has_header = kwargs.pop('has_header', False)
```

Reply:
Probably it's not necessary, service will use the default one if has_header is not specified. (Though the service default to False)